### PR TITLE
Update make-changes.mdx

### PR DESCRIPTION
**Pull request by oatsandsugar**

### CODE REVIEW

File: apps/framework-docs/src/pages/building/make-changes.mdx [hunk 1]
```diff
@@ -1,20 +1,20 @@
 # Changing your data models
-Change management in your data stack is critical to ensure that your data is always accurate and up to date based on your end user's expectations. Broken dashboards, reports, or machine learning models can lead to unhappy customers and lost revenue. MooseJS makes it easy to change your data models and ensure that your data stack is always up to date. To do so, we leverage git workflows to ensure that changes are tracked and can be rolled back if necessary.
+Change management in your data stack is critical to ensure that your data is always accurate and up to date based on your end user's expectations. Bad change management can mean broken dashboards, reports, or machine learning models can lead to unhappy customers and lost revenue. MooseJS makes it easy to change your data models and ensure that your data stack is always up to date. To do so, we leverage git workflows to ensure that changes are tracked and can be rolled back if necessary.
```
Comment (cjus) [inline apps/framework-docs/src/pages/building/make-changes.mdx:2]: I do like the change.

File: apps/framework-docs/src/pages/building/make-changes.mdx [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+Whenever you save a file, Moose will automatically detect the change and update your data stack locally. Under the hood, MooseJS manages the state of your stack and creates temporary tables each time you save your file. This allows you to test your changes without worrying about the effects on the existing data. 
 
 Once you commit your changes, MooseJS will automatically create a new version of your data stack and turn on the migration mechanisms that ensure that historical data is present in your modified schema. The migration mechanisms assume that the initial state of your data starts at the initial state of your branch. This limits the number of active migrations that are running in your stack to optimize performance and costs.
 
-If you are already using a version of the sdk, that version will remain live and be marked for deprecation. This allows you to test your changes in without needing to update all your exiting instrumentation.  
+If you are already using a version of the SDK, that version will remain live, but it will be marked for deprecation. This allows you to test your changes without needing to update all your existing instrumentation.
```
Comment (cjus) [inline apps/framework-docs/src/pages/building/make-changes.mdx:14]: Nice fixes here.

**Review by cjus — APPROVED** [review]
Nice cleanup. LGTM

**Comment by oatsandsugar** [comment]
FYI @cjus, since I'm outside the org, I'm not authorized to merge the PR

**Pull request merged**